tests/wpt/grouping_formatter.py

```diff
@@ -105,7 +105,7 @@ def build_status_line(self):
             return new_display + "No tests running.\n"
 
     def suite_start(self, data):
-        self.number_of_tests = len(data["tests"])
+        self.number_of_tests = len(data["tests"]["default"])
```

Comment:
I think a theoretically better fix is `sum(len(item) for item in data["tests"].iterkeys())`

Reply:
Thanks for the quick review! I'm assuming you meant `itervalues()` here, so I've gone ahead an updated the branch in this PR.